postcss.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  plugins: {
+    'postcss-import': {
+      root: __dirname,
+    },
+    'postcss-mixins': {},
+    'postcss-each': {},
+    'postcss-cssnext': {}
+  },

```

Comment:
CSS Modules import doesn't work because you have to specify a root in postcss config but it was recently removed so it should work with the next beta version.